ZenPacks/zenoss/OpenStack/modeler/plugins/zenoss/OpenStack.py

```diff
@@ -40,6 +40,7 @@ class OpenStack(PythonPlugin):
 
     @inlineCallbacks
     def collect(self, device, unused):
+        log.debug("Zendev test modeler collection fix zp-1234")
```

Comment:
Normally, we do not print or log the jira ticket number (ie, zp-1234).  please take that out and put it in a comment instead.  But leave the rest of the log message.  thanks